src/server/routes/welcome.ts

```diff
@@ -163,12 +163,19 @@ const OktaResendEmail = async (req: Request, res: ResponseWithRequestState) => {
 
     trackMetric('OktaWelcomeResendEmail::Failure');
 
+    const { email } = req.body;
```

Comment:
Why not move the `const { email } = req.body;` line outside the try catch block so you don't have to define it twice (on L135 and here on L166)

Reply:
Good catch!